rollup.config.js

```diff
@@ -0,0 +1,45 @@
+// plugins that we are going to use
+import babel from 'rollup-plugin-babel';
+import minify from 'rollup-plugin-babel-minify';
+import resolve from 'rollup-plugin-node-resolve';
+
+// list of plugins used during building process
+const plugins = targets => ([
+  // use Babel to transpile to ES5
+  babel({
+    // ignore node_modules/ in transpilation process
+    exclude: 'node_modules/**',
+    // ignore .babelrc (if defined) and use options defined here
+    babelrc: false,
+    // use recommended babel-preset-env without es modules enabled
+    // and with possibility to set custom targets e.g. { node: '8' }
+    presets: [['env', { modules: false, targets }]],
+    // solve a problem with spread operator transpilation https://github.com/rollup/rollup/issues/281
+    plugins: [ 'babel-plugin-transform-object-rest-spread' ],
+    // removes comments from output
+    comments: false,
```

Comment:
There's already a `.babelrc` defined -- was the intent to use _that_?   If so, perhaps it was not working b/c `.babelrc` is supposed to be legit JSON, but the one in this repo has comments.

(I think `.babelrc` will work if you move some of ^these fields into it. ...IMHO, that would make the attributes of the source code cleaner & more explicit, rather than having that intermingled w/ rollup)

Reply:
See line 12, rollup is intentionally not using them

Reply:
I see, but that still leaves the `.babelrc` file.

IMO, if you had to go with one or the other, use `.babelrc` for whatever attributes you can.

That would be one of the 1st things a developer can look at, and say "Okay, this is what the source code is like..."

...especially if ^that configuration is intermingling Babel-specific configuration with items particular to rollup?

Reply:
I guess I thought it possible library developers may want to use babel outside of rollup. In that case, rollup build process should be canonically defined all in one place, the rollup config.